lib/src/widgets/board.dart

```diff
@@ -44,6 +44,8 @@ class Chessboard extends StatefulWidget with ChessboardGeometry {
     required this.fen,
     this.opponentsPiecesUpsideDown = false,
     this.lastMove,
+    this.squareHighlights = const IMap.empty(),
```

Comment:
Per `IMap` documentation, const values must be created with the `IMapConst` constructor.

```dart
const IMap<Square, SquareHighlight> _emptyMap = IMapConst({});
```